src/streak.js

```diff
@@ -18,3 +19,17 @@ wordService.get().then((words) => {
     streakUI.paint(currStreak, totalWords, longestStreak);
   });
 });
+
+document.getElementById("hamburger").addEventListener("click", () => {
+  if (navOpen === false) {
+    navOpen = true;
+    document.getElementById("streak-main").addEventListener("click", () => {
+      document.getElementById("hamburger").click();
+    });
+  } else {
+    navOpen = false;
+    document.getElementById("streak-main").removeEventListener("click", () => {
+      click(document.getElementById("hamburger"));
+    });
+  }
+});
```

Comment:
This is a very good function, but repeating this in every javascript file seems a bit redundant. Is there a way you can share the same function over files?